telemetry/telemetrybuffer.go

```diff
@@ -327,17 +327,6 @@ func (tb *TelemetryBuffer) ConnectToTelemetryService(telemetryNumRetries, teleme
 	}
 }
 
-// TryToConnectToTelemetryService - Attempt to connect telemetry process without spawning it if it's not already running.
```

Comment:
why was this removed?

Reply:
@tamilmani1989  It was not called from any places. So, I think this is dead code. Do I miss something?